layouts/404.html

```
@@ -0,0 +1,45 @@
+  <meta name="theme-color" content="#2563eb">
+  <title>Страница не найдена &#183; {{ site.Title }}</title>
+  <meta name="description" content="Страница не найдена. Возможно, она была переименована или удалена.">
+  <link rel="canonical" href="{{ .Permalink }}">
+
+  {{/* Fonts */}}
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&family=Newsreader:ital,wght@0,400;0,500;0,600;0,700;1,400&family=JetBrains+Mono:wght@400;500;600&display=swap">
+
+  {{/* CSS */}}
+  <link rel="stylesheet" href="{{ "css/compiled/main.css" | relURL }}">
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)